ouroboros-consensus/test-consensus/Test/Dynamic/PBFT.hs

```diff
@@ -22,7 +22,6 @@ module Test.Dynamic.PBFT (
 
 import           Data.Map.Strict (Map)
 import qualified Data.Map.Strict as Map
-import           Data.Word (Word64)
```

Comment:
I would personally only include the cleanup of this file in the PR if I'm actually modifying the file.